### PR TITLE
add rbac-user plugin

### PR DESCRIPTION
**Pull request by saada**
-----

**Checklist for plugin developers:**

- [x] Read the [Plugin Naming Guide](https://github.com/GoogleContainerTools/krew/tree/master/docs/NAMING_GUIDE.md) (for new plugins)
- [x] Verify the installation from URL or a local archive works (`kubectl krew install --manifest=[...] --archive=[...]`)

### CODE REVIEW

**Comment by saada** [comment]
I signed it!

**Comment by ahmetb** [comment]
👋 Hello! Thanks for your plugin submission.

Currently I have issues with the naming as it stands today. Neither the `name: rbac-user` nor `shortDescription: Simplify user management with rbac` explain what the plugin actually does.

Can you please reconsider the `name`/`description`/`shortDescription` values while asking yourself “can I understand what this plugin does by looking at its name?”.

**Comment by saada** [comment]
Hey @ahmetb , thanks for reviewing. I update the descriptions as requested. Let me know if you still don't like the name. There's a better description here: https://github.com/saada/kubectl-rbac-user

File: plugins/rbac-user.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        matchExpressions:
+          - { key: os, operator: In, values: [darwin, linux] }
+  version: 'master'
+  shortDescription: Manage ServiceAccounts and their role bindings for easily providing user access to a cluster.
+  caveats: |
+    This plugin needs the following programs:
+    * jq
+    * bash
+  description: |
+    Easily create and delete ServiceAccounts. This plugin allows you to bind them to existing ClusterRoles in your Kubernetes cluster. The plugin simplifies the management of user access by taking care of deleting associated RoleBindings and ClusterRoleBindings associated with a ServiceAccount on deletion. After creating a user, it's easy to share a kubeconfig with the `view-serviceaccount-kubeconfig` plugin. Although it's named `rbac-user`, you can use it to also manage service access in general since it's just ServiceAccounts.
```
Comment (ahmetb) [inline plugins/rbac-user.yaml]: You can break this into multiple lines

**Comment by ahmetb** [comment]
> There's a better description here: [saada/kubectl-rbac-user](https://github.com/saada/kubectl-rbac-user)

Ideally the description should be good enough here to explain what the plugin does.

> Let me know if you still don't like the name.

It's not me not liking the name. I’m claiming right now most people won’t get what the plugin does by its name. We have other rbac plugins like

```
rbac-lookup                    Reverse lookup for RBAC                            installed
rbac-view                      A tool to visualize your RBAC permissions.         available
```

and their name is pretty clear and self-explanatory. 

After looking at your repo:
- Your plugin creates service accounts (optionally with specified role, in a specified namespace, with permissions/restrictions in other namespaces)
	- it does all these (svcacct create + bindings create) in one shot
	- similarly it deletes bindings + serviceaccounts in one shot
- I think you should probably consider adding the "Usage" section in "caveats" (or link to your documentation from "caveats").

I think your user experience should be something like:

```
kubectl rbac-binding create <svcacct> [--namespace/-n] [--role=] [...]
kubectl rbac-binding delete <svcacct> [--namespace/-n] [--role=] [...]
```

What do you think?

**Comment by saada** [comment]
I started refactoring code. Will submit update sometime this month.

**Comment by ahmetb** [comment]
I'll close this.
If you happen to iterate on the plugin to have a more clear usage or think of another plugin idea, happy to review again.
/close

**Pull request closed without merging**